src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -1122,10 +1125,6 @@ export default {
 			// TDOO: Consider extending loading state for basket updates
 			// Query to update basket state
 			this.updateBasketState();
-			// TEMPORARY: Obstruct ability to click the "Checkout" button on the loan card to prevent redirect
-			if (this.$refs.loandisplayref) {
-				this.$refs.loandisplayref.loadingLoans = true;
-			}
```

Comment:
This block should be initializing a loading overlay after/during add to basket. This is probably good to preserve...

Reply:
The button in the new KivaClassicBasicLoanCard changes the "Loan $25 now" button text to "Loading..."  Here take a look at this loom video to see the UI behavior now.  https://www.loom.com/share/5d217b088b04404da27ec588c9876f16

Reply:
Is the delay still too noticeable with the button text changing the way that it does?  I can't remember if the old loan cards did this as well or if you're just talking about the very short like <1 second delay between when the button changes to "Checkout now" and when the lightbox appears.

Reply:
Yep, the new Kiva Classic card def prevents any accidental clicking out to checkout.

The "percieved" delay of the page was an issue that got some significant number of complaints from users through SP which is why we added in the different loading states to indicate when some background process was in progress.

You could probably chat with the SP stakeholders and walk through the interface in prod to get an idea of how it's behaving and if they are ok with the current behavior...